scripts/package.json

```diff
@@ -17,7 +17,9 @@
     "underscore": "^1.8.3",
     "util": "^0.10.3"
   },
-  "devDependencies": {},
+  "devDependencies": {
+    "minimist": "^1.2.0"
```

Comment:
I guess this should be a non-dev dependency as well, since all the other ones in the scripts folder are there.

It's doesn't really matter, but 6 months from now when someone looks at this with fresh eyes they might be confused as to why one dependency is a dev dependency and the rest aren't.